### PR TITLE
Add tooltip component

### PR DESCRIPTION
**Pull request by tomrimk**
**What kind of change does this PR introduce? Explain your changes. (Bug fix, feature, docs update, ...)**

* Adds a tooltip component

**Does this close any currently open issues?**

* Solves #183 issue

**What is the new behavior (if this is a feature change)?**

You can wrap an element with a Tooltip component to make a tooltip appear on the element.
You can provide props:
* text - specifies the text that tooltip displays
* placement - specifies where the toltip should be placed (default = 'top')

```
<Tooltip text='I'm a tooltip' placement='bottom'>
  <Button></Button>
<Tooltip />
```

**Where has this been tested?**

  * Operating System: windows
  * Sidebar and player components

### CODE REVIEW

File: src/renderer/components/tooltip/view.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+class Tooltip extends React.PureComponent {
+  render() {
+    const { text, placement = 'top', children } = this.props
```
Comment (btzr-io) [inline src/renderer/components/tooltip/view.jsx]: Can you add the `placement: top` value to `defaultProps` ?

File: src/renderer/components/tooltip/view.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    return (
+      <div className="tooltip">
+        <span className={`tooltip-text tooltip-${placement}`}>{text}</span>
```
Comment (btzr-io) [inline src/renderer/components/tooltip/view.jsx]: Nice!

File: src/renderer/css/component/tooltip.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltip .tooltip-text {
+  visibility: hidden;
```
Comment (btzr-io) [inline src/renderer/css/component/tooltip.css]: do we need this? Maybe just show / hide using `display` and remove `visibility` and `opacity`?

**Comment by btzr-io** [comment]
Awesome, thanks for your contribution, looks like there is already a `pr` for the same issue,
If the first solution can't be merged, this pull request will be :+1:

File: src/renderer/components/tooltip/view.jsx (hunk 1 above)
Comment (tomrimk) [reply src/renderer/components/tooltip/view.jsx]: Added. Can you check it?

File: src/renderer/css/component/tooltip.css (hunk 3 above)
Comment (tomrimk) [reply src/renderer/css/component/tooltip.css]: Probably not. Changed it to your suggestion.

**Pull request merged**